homeassistant/util/collection.py

```diff
@@ -0,0 +1,36 @@
+"""Helpers for working with collections."""
+
+from collections.abc import Collection, Iterable
+from functools import partial
+from itertools import islice
+from typing import Any
+
+
```

Comment:
Let's do the breakout in a separate PR

Reply:
https://github.com/home-assistant/core/pull/118065